crates/control/src/motion/command_sender.rs

```diff
@@ -17,10 +17,10 @@ pub struct CreationContext {}
 pub struct CycleContext {
     optimized_motor_commands: Input<MotorCommands<Joints<f32>>, "optimized_motor_commands">,
     leds: Input<Leds, "leds">,
-
+    joint_calibration_offsets: Parameter<Joints<f32>, "joint_calibration_offsets">,
     motion_safe_exits: CyclerState<MotionSafeExits, "motion_safe_exits">,
-    last_actuated_motor_commands:
-        CyclerState<MotorCommands<Joints<f32>>, "last_actuated_motor_commands">,
+    last_actuated_motor_commands_before_offset:
```

Comment:
As you are now definitely the last who adds the offsets, there is no way for the user to access any joint angles after the offsets, so the renaming should stay the same as in main

```suggestion
    last_actuated_motor_commands:
```

Reply:
true
done